src/components/Events.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import type { Event } from "../model/Event";
+import style from "./Events.module.scss";
+import { Loader } from "../components/Loader";
+
+export const Events: React.FC = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const response = await fetch(
+          "https://sportujspolu-api.onrender.com/api/v1/events"
+        );
+        setEvents(await response.json());
+        setIsLoading(false);
+      } catch (error) {}
+    };
+
+    fetchEvents();
+  }, []);
+
+  return (
+    <div>
+      {isLoading ? (
+        <Loader />
+      ) : (
+        <ul className={style.list}>
```

Comment:
Making huge ternaries in the return can make the JSX very hard to read. People who don't know what they are doing love it.

I prefer early edge case return. So, in this case, do:
```suggestion

if(isLoading) {
 return <Loader />
}


return
        <ul className={style.list}>
```

if the ternary is somewhere deep in the JSX, it might not be the best pattern.  Small ternaries are ok. It is really about personal preference

Reply:
updated